node/commons/compatibility-layer.ts

```diff
@@ -24,71 +18,178 @@ export const convertBiggyProduct = (
       })
     : []
 
-  const skus: any[] = (product.skus || []).map(
+
+  const categoriesIds: string[] = product.categoryIds
+  ? product.categoryIds.map((_: any, index: number) => {
+      const subArray = product.categoryIds.slice(0, index + 1)
+      return `/${subArray.join('/')}/`
+    }).reverse()
+  : []
+
+  const skus: SearchItem[] = (product.skus || []).map(
     convertSKU(product, indexingType, tradePolicy)
   )
 
+  const allSpecifications = product.productSpecifications.concat(getSKUSpecifications(product))
+
+  const specificationGroups = JSON.parse(product.specificationGroups)
+
+  const allSpecificationsGroups = [ ...Object.keys(specificationGroups) ]
+
+  const brandId = product.brandId ? Number(product.brandId) : -1
```

Comment:
Is it possible that the `brandId` is `0`? If it is, the `brandId` will become `-1` since the number `0` is falsy.

Reply:
As far as I'm aware, no. All stores I've checked that have only 1 brand start on brandId `1`